File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/PatientLinkingFwkDoctorInfo.java
package nta.med.data.model.ihis.ocsa;

public class PatientLinkingFwkDoctorInfo {

	private String doctorCode;
	private String doctorName;

	public PatientLinkingFwkDoctorInfo(String doctorCode, String doctorName) {
		super();
		this.doctorCode = doctorCode;
		this.doctorName = doctorName;
	}

	public String getDoctorCode() {
		return doctorCode;
	}

	public void setDoctorCode(String doctorCode) {
		this.doctorCode = doctorCode;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/PatientLinkingFwkDoctorInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class PatientLinkingFwkDoctorInfo
    {
        private String _doctorCode;
        private String _doctorName;

        public String DoctorCode
        {
            get { return this._doctorCode; }
            set { this._doctorCode = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public PatientLinkingFwkDoctorInfo() { }

        public PatientLinkingFwkDoctorInfo(String doctorCode, String doctorName)
        {
            this._doctorCode = doctorCode;
            this._doctorName = doctorName;
        }

    }
}